### DISCUSSION TITLE
Management page for file attachments?

### DISCUSSION THREAD

**Discussion by michaelrsweet** [community/community#8547] [Projects and Issues]
It seems strange that people can attach random files to issues, discussions, pull requests, and releases but the repository admins cannot manage those files short of asking Github support to remove files that shouldn't be there?!?

The basic functionality I'm looking for is:

- List of files associated with project
- Ability to delete those files with a suitable "are you sure" barrier

Nice to have would be a way to see where each of the files is referenced, although I understand that might be harder... :)

**Comment by codeling** [comment]
This now seems to be actively exploited by malware (see e.g. https://www.bleepingcomputer.com/news/security/github-comments-abused-to-push-malware-via-microsoft-repo-urls/ or https://www.schneier.com/blog/archives/2024/04/using-legitimate-github-urls-for-malware.html)

github, are you planning do do anything about this? or will it be necessary  to disable issues and PRs explicitly in every repository in order to avoid this? I would prefer an option to disable users from uploading any attachments in issues and PRs...
